### ISSUE TITLE
Bcrypt fail - can't find Python executable

### ISSUE THREAD

**Issue by RavDanny** [CyCoreSystems/docker-meteor#55]
Get the following error using `ulexus/meteor:build` - I included the full stdout:
```
Cloning https://...
Cloning into '/home/meteor/src'...
Switching to branch/tag master...
Already on 'master'
Your branch is up-to-date with 'origin/master'.
Forcing clean...
HEAD is now at 5a4dc06 display analysis error page
Meteor source found in /home/meteor/src/myapp/.meteor
Checking Meteor version...
/usr/bin/entrypoint.sh: line 93: [: -Z: unary operator expected
Application's Meteor version (1.5) is less than 1.3.1; please use ulexus/meteor:legacy
Downloading Meteor install script...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7782    0  7782    0     0   6727      0 --:--:--  0:00:01 --:--:--  6731
Installing Meteor 1.5...
Downloading Meteor distribution

Meteor 1.5 has been installed in your home directory (~/.meteor).
Writing a launcher script to /usr/local/bin/meteor for your convenience.

To get started fast:

  $ meteor create ~/my_cool_app
  $ cd ~/my_cool_app
  $ meteor

Or see the docs at:

  docs.meteor.com

Installing application-side NPM dependencies...
npm WARN deprecated gcloud@0.10.0: gcloud has been renamed to google-cloud. To get new features and bug fixes, you must use the new package.
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead

> bcrypt@0.8.7 install /home/meteor/src/pals/node_modules/bcrypt
> node-gyp rebuild

gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
gyp ERR! stack     at Object.failNoPython (/home/meteor/.meteor/packages/meteor-tool/.1.5.0.utbu0o++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/node-gyp/lib/configure.js:482:19)
gyp ERR! stack     at Object.<anonymous> (/home/meteor/.meteor/packages/meteor-tool/.1.5.0.utbu0o++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/node-gyp/lib/configure.js:396:16)
gyp ERR! stack     at F (/home/meteor/.meteor/packages/meteor-tool/.1.5.0.utbu0o++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/which/which.js:68:16)
gyp ERR! stack     at E (/home/meteor/.meteor/packages/meteor-tool/.1.5.0.utbu0o++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/which/which.js:80:29)
gyp ERR! stack     at /home/meteor/.meteor/packages/meteor-tool/.1.5.0.utbu0o++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/which/which.js:89:16
gyp ERR! stack     at /home/meteor/.meteor/packages/meteor-tool/.1.5.0.utbu0o++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /home/meteor/.meteor/packages/meteor-tool/.1.5.0.utbu0o++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:82:15)
gyp ERR! System Linux 4.4.0-45-generic
gyp ERR! command "/home/meteor/.meteor/packages/meteor-tool/.1.5.0.utbu0o++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node" "/home/meteor/.meteor/packages/meteor-tool/.1.5.0.utbu0o++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/meteor/src/pals/node_modules/bcrypt
gyp ERR! node -v v4.8.3
gyp ERR! node-gyp -v v3.6.0
gyp ERR! not ok
npm WARN pals No description
npm WARN/home/meteor/src/pals
+-- babel-runtime@6.26.0
| +-- core-js@2.5.1
| `-- regenerator-runtime@0.11.0
+-- pkgcloud@1.5.0
| +-- async@0.9.2
| +-- aws-sdk@2.142.0
| | +-- buffer@4.9.1
| | | +-- base64-js@1.2.1
| | | +-- ieee754@1.1.8
| | | `-- isarray@1.0.0
| | +-- crypto-browserify@1.0.9
| | +-- events@1.1.1
| | +-- jmespath@0.15.0
| | +-- querystring@0.2.0
| | +-- sax@1.2.1
| | +-- url@0.10.3
| | | +-- punycode@1.3.2
| | | `-- querystring@0.2.0  deduped
| | +-- uuid@3.1.0
| | +-- xml2js@0.4.17
| | | +-- sax@1.2.1  deduped
| | | `-- xmlbuilder@4.2.1  deduped
| | `-- xmlbuilder@4.2.1
| |   `-- lodash@4.17.4
| +-- errs@0.3.2
| +-- eventemitter2@0.4.14
| +-- fast-json-patch@0.5.7
| +-- filed@0.1.0
| | `-- mime@1.4.1  deduped
| +-- gcloud@0.10.0
| | +-- duplexify@3.5.1
| | | +-- end-of-stream@1.4.0
| | | | `-- once@1.4.0
| | | |   `-- wrappy@1.0.2
| | | +-- inherits@2.0.3
| | | +-- readable-stream@2.3.3
| | | | +-- core-util-is@1.0.2  deduped
| | | | +-- inherits@2.0.3  deduped
| | | | +-- isarray@1.0.0  deduped
| | | | +-- process-nextick-args@1.0.7
| | | | +-- safe-buffer@5.1.1  deduped
| | | | +-- string_decoder@1.0.3
| | | | | `-- safe-buffer@5.1.1  deduped
| | | | `-- util-deprecate@1.0.2
| | | `-- stream-shift@1.0.0
| | +-- extend@1.3.0
| | +-- gapitoken@0.1.5
| | | +-- jws@3.0.0
| | | | +-- base64url@1.0.6
| | | | | +-- concat-stream@1.4.10
| | | | | | +-- inherits@2.0.3  deduped
| | | | | | +-- readable-stream@1.1.14
| | | | | | | +-- core-util-is@1.0.2  deduped
| | | | | | | +-- inherits@2.0.3  deduped
| | | | | | | +-- isarray@0.0.1
| | | | | | | `-- string_decoder@0.10.31
| | | | | | `-- typedarray@0.0.6
| | | | | `-- meow@2.0.0
| | | | |   +-- camelcase-keys@1.0.0
| | | | |   | +-- camelcase@1.2.1
| | | | |   | `-- map-obj@1.0.1
| | | | |   +-- indent-string@1.2.2
| | | | |   | +-- get-stdin@4.0.1
| | | | |   | +-- minimist@1.2.0  deduped
| | | | |   | `-- repeating@1.1.3
| | | | |   |   `-- is-finite@1.0.2
| | | | |   |     `-- number-is-nan@1.0.1
| | | | |   +-- minimist@1.2.0
| | | | |   `-- object-assign@1.0.0
| | | | `-- jwa@1.0.2
| | | |   +-- base64url@0.0.6
| | | |   +-- buffer-equal-constant-time@1.0.1
| | | |   `-- ecdsa-sig-formatter@1.0.9
| | | |     +-- base64url@2.0.0
| | | |     `-- safe-buffer@5.1.1  deduped
| | | `-- request@2.83.0  deduped
| | +-- node-uuid@1.4.8
| | +-- protobufjs@3.8.2
| | | +-- ascli@0.3.0
| | | | +-- colour@0.7.1
| | | | `-- optjs@3.2.2
| | | `-- bytebuffer@3.5.5
| | |   +-- bufferview@1.0.1
| | |   `-- long@2.4.0
| | +-- request@2.83.0  deduped
| | +-- stream-events@1.0.2
| | | `-- stubs@3.0.0
| | `-- through2@0.6.5  deduped
| +-- ip@0.3.3
| +-- lodash@3.10.1
| +-- mime@1.4.1
| +-- qs@1.2.2
| +-- request@2.83.0
| | +-- aws-sign2@0.7.0
| | +-- aws4@1.6.0
| | +-- caseless@0.12.0
| | +-- combined-stream@1.0.5
| | | `-- delayed-stream@1.0.0
| | +-- extend@3.0.1
| | +-- forever-agent@0.6.1
| | +-- form-data@2.3.1
| | | +-- asynckit@0.4.0
| | | +-- combined-stream@1.0.5  deduped
| | | `-- mime-types@2.1.17  deduped
| | +-- har-validator@5.0.3
| | | +-- ajv@5.3.0
| | | | +-- co@4.6.0
| | | | +-- fast-deep-equal@1.0.0
| | | | +-- fast-json-stable-stringify@2.0.0
| | | | `-- json-schema-traverse@0.3.1
| | | `-- har-schema@2.0.0
| | +-- hawk@6.0.2
| | | +-- boom@4.3.1
| | | | `-- hoek@4.2.0  deduped
| | | +-- cryptiles@3.1.2
| | | | `-- boom@5.2.0
| | | |   `-- hoek@4.2.0  deduped
| | | +-- hoek@4.2.0
| | | `-- sntp@2.1.0
| | |   `-- hoek@4.2.0  deduped
| | +-- http-signature@1.2.0
| | | +-- assert-plus@1.0.0
| | | +-- jsprim@1.4.1
| | | | +-- assert-plus@1.0.0  deduped
| | | | +-- extsprintf@1.3.0
| | | | +-- json-schema@0.2.3
| | | | `-- verror@1.10.0
| | | |   +-- assert-plus@1.0.0  deduped
| | | |   +-- core-util-is@1.0.2  deduped
| | | |   `-- extsprintf@1.3.0  deduped
| | | `-- sshpk@1.13.1
| | |   +-- asn1@0.2.3
| | |   +-- assert-plus@1.0.0  deduped
| | |   +-- bcrypt-pbkdf@1.0.1
| | |   | `-- tweetnacl@0.14.5  deduped
| | |   +-- dashdash@1.14.1
| | |   | `-- assert-plus@1.0.0  deduped
| | |   +-- ecc-jsbn@0.1.1
| | |   | `-- jsbn@0.1.1  deduped
| | |   +-- getpass@0.1.7
| | |   | `-- assert-plus@1.0.0  deduped
| | |   +-- jsbn@0.1.1
| | |   `-- tweetnacl@0.14.5
| | +-- is-typedarray@1.0.0
| | +-- isstream@0.1.2
| | +-- json-stringify-safe@5.0.1
| | +-- mime-types@2.1.17
| | | `-- mime-db@1.30.0
| | +-- oauth-sign@0.8.2
| | +-- performance-now@2.1.0
| | +-- qs@6.5.1
| | +-- safe-buffer@5.1.1
| | +-- stringstream@0.0.5
| | +-- tough-cookie@2.3.3
| | | `-- punycode@1.4.1
| | +-- tunnel-agent@0.6.0
| | | `-- safe-buffer@5.1.1  deduped
| | `-- uuid@3.1.0  deduped
| +-- s3-upload-stream@1.0.7
| +-- through2@0.6.5
| | +-- readable-stream@1.0.34
| | | +-- core-util-is@1.0.2
| | | +-- inherits@2.0.3  deduped
| | | +-- isarray@0.0.1
| | | `-- string_decoder@0.10.31
| | `-- xtend@4.0.1
| +-- url-join@0.0.1
| `-- xml2js@0.1.14
|   `-- sax@1.2.1  deduped
`-- redis@2.8.0
  +-- double-ended-queue@2.1.0-0
  +-- redis-commands@1.3.1
  `-- redis-parser@2.6.0

 pals No repository field.
npm WARN pals No license field.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bcrypt@0.8.7 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bcrypt@0.8.7 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/meteor/.meteor/packages/meteor-tool/1.5.0/mt-os.linux.x86_64/dev_bundle/.npm/_logs/2017-11-02T05_12_49_291Z-debug.log
```

Do I need to configure the PYTHON variable - or - is it a bug?

Thanks

**Comment by Ulexus** [comment]
Well that's irritating.  I don't know if python is a new requirement or if the base image no longer includes it.  Either way, I'll get it added.

**Comment by Ulexus** [comment]
Fixed by 43a7c70e5ec3442082fe7c64bafc5c10c5d3b14c

**Comment by RavDanny** [comment]
Thanks
